src/tracks/gosling-track/gosling-track-model.ts

```diff
@@ -115,10 +115,6 @@ export class GoslingTrackModel {
         return this.id;
     }
 
-    public getRenderingId(): string {
-        return this.spec()._renderingId ?? this.getId();
-    }
-
     public originalSpec(): SingleTrack {
```

Comment:
I assume the changes in this file are that unused functions are removed.

Reply:
Exactly! Knip identified these.